xonsh/aliases.py

```diff
@@ -13,7 +13,7 @@
 from xonsh.environ import locate_binary, make_args_env
 from xonsh.foreign_shells import foreign_shell_data
 from xonsh.jobs import jobs, fg, bg, clean_jobs
-from xonsh.platform import ON_ANACONDA, ON_DARWIN, ON_WINDOWS, ON_FREEBSD, ON_NETBSD
+from xonsh.platform import ON_ANACONDA, ON_DARWIN, ON_WINDOWS, ON_FREEBSD, ON_NETBSD, ON_DRAGONFLY
```

Comment:
Ahh, and I think `black` is unhappy about the length of this line -- can you run `black` on `aliases.py` to fix it?  Thanks!

Reply:
Eh, so for my OS py36-black pulled in like 4GB of deps and then it was broken and I had to install something else to fix it, got it up, but it seems that the lint also changed more than just that line. If you'd prefer I can take those changes back out, but I left them for the moment. Your call.

Reply:
Sorry about the black issue, that seems uncharacteristically large. I'll fix the remaining black issue